### PR TITLE
Austin lab10

### PR DESCRIPTION
**Pull request by overcommit**
Submitting lab for review.

### CODE REVIEW

**Review by justin-young5 — COMMENTED** [review]
There are two versions for this lab, make sure you include version 2 on this branch and push it for grading.  Thanks.

**Comment by overcommit** [comment]
> There are two versions for this lab, make sure you include version 2 on this branch and push it for grading. Thanks.

Added v2

**Review by violette-git — APPROVED** [review]
Great work. Very clean. Just need to add the exit functionality.

File: code/AustinD/lab10v2.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  quotes = get_quotes(page, keyword)
+
+  while True:
+    for quote in quotes['quotes']:
+      author, body = parse_quotes(quote)
+      print(f"{body}\n{author}")
+
+    next_page = input("Enter 'next page' or 'done':")
+    if next_page.lower() == 'done':
+      keyword = input("Enter a new keyword:")
```
Comment (violette-git) [inline code/AustinD/lab10v2.py:27]: I do think this was a mistake here. We want to end the program when the user inputs 'done'

**Review by violette-git — CHANGES_REQUESTED** [review]

File: code/AustinD/lab10v2.py (hunk 1 above)
Comment (overcommit) [reply code/AustinD/lab10v2.py]: Hi, sorry, misunderstood that part of the assignment, easy fix just did it.

**Review by violette-git — APPROVED** [review]

**Pull request merged**